### PR TITLE
Add JavaScript Object compat data

### PR DESCRIPTION
**Pull request by Elchi3**

### CODE REVIEW

File: javascript/builtins/Object.json [hunk 1]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": true,
+                "version_removed": "43"
+              },
+              "firefox_android": {
+                "version_added": false
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Should match Firefox here.

File: javascript/builtins/Object.json [hunk 2]
```diff
@@ -0,0 +1,2444 @@
...
+              "deprecated": false
+            }
+          }
+        },
+        "assign": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object/assign",
+            "support": {
+              "webview_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: `false` according to the data? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility

File: javascript/builtins/Object.json [hunk 3]
```diff
@@ -0,0 +1,2444 @@
...
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "32"
+              },
+              "opera_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: `false` according to the data? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility

File: javascript/builtins/Object.json [hunk 4]
```diff
@@ -0,0 +1,2444 @@
...
+                "version_added": "9"
+              },
+              "ie_mobile": {
+                "version_added": true
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "10.6"
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.6 according to the data: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create#Browser_compatibility

File: javascript/builtins/Object.json [hunk 5]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "opera_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.5 according to the data: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create#Browser_compatibility

File: javascript/builtins/Object.json [hunk 6]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 4 according to the data: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create#Browser_compatibility

File: javascript/builtins/Object.json [hunk 7]
```diff
@@ -0,0 +1,2444 @@
...
+                "version_added": "9"
+              },
+              "ie_mobile": {
+                "version_added": true
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "10.6"
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.6 according to the data

File: javascript/builtins/Object.json [hunk 8]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "opera_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.5 according to the data

File: javascript/builtins/Object.json [hunk 9]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.4 according to the data

File: javascript/builtins/Object.json [hunk 10]
```diff
@@ -0,0 +1,2444 @@
...
+                "notes": "Also supported in Internet Explorer 8, but only on DOM objects and with some non-standard behaviors."
+              },
+              "ie_mobile": {
+                "version_added": true
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "10.6"
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.6 according to the data.

File: javascript/builtins/Object.json [hunk 11]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "opera_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 11.5 according to the data.

File: javascript/builtins/Object.json [hunk 12]
```diff
@@ -0,0 +1,2444 @@
...
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: 4 according to the data.

File: javascript/builtins/Object.json [hunk 13]
```diff
@@ -0,0 +1,2444 @@
...
+              "deprecated": false
+            }
+          }
+        },
+        "freeze": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze",
+            "support": {
+              "webview_android": {
+                "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Some of the mobile browsers are Yes here and ? in the tables, but I assume that is intentional :)

File: javascript/builtins/Object.json [hunk 14]
```diff
@@ -0,0 +1,2444 @@
...
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Table says 23, FWIW.

File: javascript/builtins/Object.json [hunk 15]
```diff
@@ -0,0 +1,2444 @@
...
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptors",
+            "support": {
+              "webview_android": {
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": "54"
+              },
+              "chrome_android": {
+                "version_added": null
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Does Chrome Android usually match Chrome?

File: javascript/builtins/Object.json [hunk 16]
```diff
@@ -0,0 +1,2444 @@
...
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Marked deprecated in the docs page.

File: javascript/builtins/Object.json [hunk 17]
```diff
@@ -0,0 +1,2444 @@
...
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Also marked deprecated in the docs.

File: javascript/builtins/Object.json [hunk 18]
```diff
@@ -0,0 +1,2444 @@
...
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Also marked deprecated.

File: javascript/builtins/Object.json [hunk 19]
```diff
@@ -0,0 +1,2444 @@
...
```
Comment (wbamberg) [inline javascript/builtins/Object.json]: Also marked deprecated.

File: javascript/builtins/Object.json (hunk 6 above)
Comment (Elchi3) [reply javascript/builtins/Object.json]: Interesting! We really should work on version validation. So far I thought that version Firefox for Android version 4 is not really existing, but in fact it is the very first one according to https://en.wikipedia.org/wiki/Firefox_for_Android#Release_history

File: javascript/builtins/Object.json (hunk 9 above)
Comment (Elchi3) [reply javascript/builtins/Object.json]: Firefox for Android and 11.4? I guess GitHub screwed up the line numbers here or something.

File: javascript/builtins/Object.json (hunk 13 above)
Comment (Elchi3) [reply javascript/builtins/Object.json]: Yeah, this should be supported (ES 5.1)

File: javascript/builtins/Object.json (hunk 14 above)
Comment (Elchi3) [reply javascript/builtins/Object.json]: This is totally obsolete and is not supported anymore. Not sure it is worth finding out the support range when it was supported. (23 to ?). I'll leave this as false.

File: javascript/builtins/Object.json (hunk 15 above)
Comment (Elchi3) [reply javascript/builtins/Object.json]: I think for the newer versions, yes.

**Comment by Elchi3** [comment]
PTAL :)

File: javascript/builtins/Object.json (hunk 9 above)
Comment (wbamberg) [reply javascript/builtins/Object.json]: No, I screwed up the copy/pasting :). I meant 4.

**Pull request merged**